Plate_2_data/2.cellprofiler_ic_processing/loaddata_edit_and_ic.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<p>Consider spelling out "illumination" as some people may not know what <code>illum</code> means.</p><p> </p><p>I've notice <code>illum</code> is used multiple times. Consider spelling it out fully once and then shorten it. For example <code>illuminiation (illum)</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>